api/backend_test.go

```diff
@@ -614,3 +614,79 @@ func TestLoginWithKey(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, crypto.PubkeyToAddress(chatKey.PublicKey), extkey.Address)
 }
+
+func TestDeleteMulticcount(t *testing.T) {
+	backend := NewGethStatusBackend()
+
+	rootDataDir, err := ioutil.TempDir("", "test-keystore-dir")
```

Comment:
Error checking, also run `defer` as soon as you check for the error, otherwise if `require.NoError(t,err)` fails on line 623, it won't run it.

Reply:
fixed, thanks